### PR TITLE
i18n(fr): Update `resources/community-content.mdx` from #2049

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description
 Update `resources/community-content.mdx` from #2049
- i18n(fr)

### CODE REVIEW

**Review by pioupia — APPROVED** [review]
LGTM!

File: docs/src/content/docs/fr/resources/community-content.mdx [hunk 1]
```diff
@@ -127,24 +127,27 @@ import YouTubeGrid from '~/components/youtube-grid.astro';
...
-				"J'ai recréé la documentation de Shadcn en 30 minutes avec cet outil incroyable",
+			title: "J'ai recréé la documentation de Shadcn en 30 minutes avec cet outil incroyable",
 			description:
 				"Dans cette vidéo, je passe en revue ce qui rend Starlight si cool et pourquoi vous voudriez l'essayer avec votre prochain projet.",
 		},
+		{
+			href: 'https://www.youtube.com/watch?v=Q1E4Gkt63ko',
+			title: "Astro Starlight et emoji-blast avec Chris Swithinbank",
+			description:
+				"Chris, le créateur du framework Astro's Starlight, explique comment le mettre en place pour un nouveau site web d'emoji-blast. 🌟 🎇",
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/resources/community-content.mdx:150]:
```suggestion
				"Chris, le créateur du thème de documentation Starlight pour Astro, m'aide à le configurer pour le nouveau site web d'emoji-blast. 🌟 🎇",
```

The English `'s` in `Astro's Starlight` was remaining so I tweaked the sentence a bit to remove it and uses the same description ("thème de documentation") that we are using on the French [version](https://starlight.astro.build/getting-started/) of the getting started guide.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the update 🙌 

I tweaked the sentence a little bit to remove the remaining English `'s`. Let me know what you think about the change.

**Comment by thomasbnt** [comment]
> Thanks for the update 🙌
> 
> I tweaked the sentence a little bit to remove the remaining English `'s`. Let me know what you think about the change.

Thanks for the review @HiDeoo !

**Review by HiDeoo — APPROVED** [review]
Amazing, thanks for the translation and the review :star2:

**Pull request merged**